src/data/gitmojis.json

```
@@ -235,7 +235,7 @@
       "emoji": "🚚",
       "entity": "&#1F69A;",
       "code": ":truck:",
-      "description": "Move or rename files.",
+      "description": "Rename or move resources (files, paths, routes).",
```

Why not keep the original order?

```suggestion
      "description": "Move or rename resources (files, paths, routes).",
```

---

ops :sweat_smile: